docs/externalGrading.md

```diff
@@ -238,36 +238,58 @@ Running PrairieLearn locally with externally graded question support looks somet
+- Modify your PL docker run call to include the jobs directory.
+
+On MacOS and Linux, `cd` to your course directory and copy-paste the following command:
 ```sh
 docker run -it --rm -p 3000:3000 \
-    -v ${PWD}:/course `# Map your current directly in as course content` \
-    -v ${HOME}/pl_ag_jobs:/jobs `# Map jobs directory into /jobs` \
-    -e HOST_JOBS_DIR=$HOME/pl_ag_jobs \
+    -v "$PWD":/course `# Map your current directory in as course content` \
+    -v "$HOME/pl_ag_jobs:/jobs" `# Map jobs directory into /jobs` \
+    -e HOST_JOBS_DIR="$HOME/pl_ag_jobs" \
     -v /var/run/docker.sock:/var/run/docker.sock `# Mount docker into itself so container can spawn others` \
     prairielearn/prairielearn 
 ```
 
-**Note:** The sequence above is verified to work on Windows 10 as well as MacOS and Linux docker. 
+On Windows PowerShell, `cd` to your course directory and copy the following command **except** with your own username in `HOST_JOBS_DIR`:
+```sh
+docker run -it --rm -p 3000:3000 -v $PWD\:/course -v $HOME\pl_ag_jobs:/jobs -e HOST_JOBS_DIR=/c/Users/Tim/pl_ag_jobs -v /var/run/docker.sock:/var/run/docker.sock prairielearn/prairielearn 
+```
+
+**Note** the following about `HOST_JOBS_DIR` on PowerShell:
+
+* Use Unix-style paths (i.e., use `/c/Users/Tim/pl_ag_jobs`, **not** `C:\Users\Tim\pl_ag_jobs`)
+* Use the full path rather than `$HOME` (i.e., use `/c/Users/Tim/pl_ag_jobs`, **not** `$HOME/pl_ag_jobs`).
 
-##### Common issues
+##### Windows errors and quirks
 
-Windows users might see problems running grading jobs. The most common issue is the:
+Windows users might see problems running grading jobs:
 
 ```bash
 standard_init_linux.go:207: exec user process caused "no such file or directory"
 ```
 
-The error occurs during grading as a result of an OS new-line incompatibility with the `entrypoint` script in the externally
+This error occurs during grading as a result of an OS new-line incompatibility with the `entrypoint` script in the externally
 graded question. One solution for this is to make a `.gitattributes` files in your PL repository with the line
 `*.sh text eol=lf`. This tells the GitHub client to write the script files in native Linux
 format instead of converting them for Windows (which breaks mapping them back into docker). 
 This mimics the [`.gitattributes` file in the main PrairieLearn repo](https://github.com/PrairieLearn/PrairieLearn/blob/master/.gitattributes).
 
+
+```sh
+error: Error processing external grading job 1
+error: handleGraderErrorUnable to launch Docker container for grading: (HTTP code 500) server error - invalid mode: /grade
+```
+
+This error can occur when `HOST_JOBS_DIR` cannot be accessed.
+
+1. Verify that the `pl_ag_jobs` directory was created successfully.
+2. Verify that you are using Unix-style slashes even though you are using PowerShell (i.e., use `-e HOST_JOBS_DIR=/c/Users/Tim/pl_ag_jobs`, **not** `-e HOST_JOBS_DIR=C:\Users\Tim\pl_ag_jobs`).
```

Comment:
I think that this error also occurs if Docker hasn't got write/share/something access to the disk containing `HOST_JOBS_DIR`. Is that right? If so, the fix for that might be rebooting, restarting Docker, or re-doing the Docker disk sharing permissions in the Docker config dialog.

Reply:
Added